src/System.Management.Automation/resources/FileSystemProviderStrings.resx

```diff
@@ -232,7 +232,7 @@
     <value>The property {0} does not exist or was not found.</value>
   </data>
   <data name="PermissionError" xml:space="preserve">
-    <value>You do not have sufficient access rights to perform this operation.</value>
+    <value>You do not have sufficient access rights to perform this operation or the item is hidden, system or read only.</value>
```

Comment:
Do we want to say try `-Force`?

```suggestion
    <value>You do not have sufficient access rights to perform this operation or the item is hidden, system, or read only.</value>
```

Reply:
Isn't it a task for suggestion subsystem?